neo/Core/TX/test_transactions.py

```diff
@@ -203,3 +204,16 @@ def test_tx_big_remark(self):
             tx = Transaction.DeserializeFrom(reader)
 
             self.assertEqual(tx.Hash.ToString(), self.giant_tx_hash)
+
+    def test_GetScriptHashesForVerifying_invalid_operation(self):
+        with patch("neo.Core.Helper", return_value=None):
+            with patch("neo.Blockchain", return_value=None):
+                with self.assertRaises(Exception) as e:
```

Comment:
the patching isn't needed, because in this test the `LevelDBBlockchain` isn't used, but the abstract `Blockchain` class, so `asset = GetBlockchain().GetAssetState(key.ToBytes())` points to 
https://github.com/CityOfZion/neo-python/blob/e002fa580388aedb6b83ef8fd8a4a2867b858813/neo/Core/Blockchain.py#L304-L306
which will always be `None` for
https://github.com/CityOfZion/neo-python/blob/e002fa580388aedb6b83ef8fd8a4a2867b858813/neo/Core/TX/Transaction.py#L719-L721

If the idea is to increase coverage, then why don't we provide an asset that really does not exist?

Reply:
Mostly because I thought mocking output was easier because I could start with a `tx` that was already built in the test. Also, originally, I was trying to include the test of `build_raw_transactions.py` so every part would have been covered except this. I will see what I can do.